exporter/redis.go

```diff
@@ -370,15 +370,26 @@ func parseConnectedSlaveString(slaveName string, slaveInfo string) (offset float
 	return
 }
 
-func extractConfigMetrics(config []string, addr string, alias string, scrapes chan<- scrapeResult) error {
+func extractConfigMetrics(config []string, addr string, alias string, scrapes chan<- scrapeResult) (dataBasesCount int, err error) {
 	if len(config)%2 != 0 {
-		return fmt.Errorf("invalid config: %#v", config)
+		err = fmt.Errorf("invalid config: %#v", config)
+		return
 	}
 
 	for pos := 0; pos < len(config)/2; pos++ {
 		strKey := config[pos*2]
 		strVal := config[pos*2+1]
 
+		if strKey == "databases" {
+			var err error
```

Comment:
Last nitpick: you don't need this any longer as you now declare it up in line 373 and you can shorten the next few lines to
```
if dbCount, err = strconv.Atoi(strVal); err != nil {
  return 0, fmt.Errorf("invalid config value for key databases: %#v", strVal)
}
```

Reply:
used your recommendation & shortened